Microsoft.Toolkit.Uwp.UI.Controls/Microsoft.Toolkit.Uwp.UI.Controls.csproj

```diff
@@ -11,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="$(MSBuildThisFileDirectory)readme.txt" Pack="true" PackagePath="\"/>
+    <None Include="$(MSBuildThisFileDirectory)ReadMe.md" Pack="true" PackagePath="\" />
```

Comment:
❗ Please don't rename this file, this is explicitly read in as `readme.txt` for Visual Studio to load.

https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#add-a-readme-and-other-files

Reply:
I thought markdown format would be a better experience for users.

Visual Studio can load markdown file too, right? I think I've seen it on some packages.

I'll revert the change if it doesn't work. Is that okay with you?

Reply:
Seems markdown file doesn't work. Reverting back to text file type.